Add Captain artwork by Mehdi-BR

Mehdi-BR opened the pull request.

MarkEWaite reviewed (approved):
Thanks @Mehdi-BR .  I trimmed 5 pixels on each side of the image you named `265.png` with imagemagick and then used imagemagick scale to make it 256 pixels wide like the other images used for the artwork page.  I modified the file names to not include the word "Jenkins" because that is implied for all the artwork.  I switched to lower case characters in the names for consistency with the other artwork.

See my changes in b912a1a67 and see them in context on the [preview page](https://deploy-preview-5363--jenkins-io-site-pr.netlify.app/artwork/).

I think this is ready to merge.  If you respond to confirm that, I'll merge it as soon as I see your message.  If I don't see your response within the next 24 hours, I'll merge it tomorrow

Mehdi-BR commented:
Hello Mark

Indeed I would like you to merge it please.
And thank you.

Le dim. 14 août 2022 à 15:33, Mark Waite ***@***.***> a
écrit :

> ***@***.**** approved this pull request.
>
> Thanks @Mehdi-BR <https://github.com/Mehdi-BR> . I trimmed 5 pixels on
> each side of the image you named 265.png with imagemagick and then used
> imagemagick scale to make it 256 pixels wide like the other images used for
> the artwork page. I modified the file names to not include the word
> "Jenkins" because that is implied for all the artwork. I switched to lower
> case characters in the names for consistency with the other artwork.
>
> See my changes in b912a1a
> <https://github.com/jenkins-infra/jenkins.io/commit/b912a1a672a84ba24cd4acf2c74de3c833c3464c>
> and see them in context on the preview page
> <https://deploy-preview-5363--jenkins-io-site-pr.netlify.app/artwork/>.
>
> I think this is ready to merge. If you respond to confirm that, I'll merge
> it as soon as I see your message. If I don't see your response within the
> next 24 hours, I'll merge it tomorrow
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jenkins-infra/jenkins.io/pull/5363#pullrequestreview-1072091653>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANQ45CMFKGJOJFTP3GFIWBLVZD7UZANCNFSM56P2SNXA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
